infer/src/bo/basicDom.ml

```diff
@@ -0,0 +1,91 @@
+(*
```

Comment:
A less generic filename would be useful here. Otherwise, it might be hard to tell which analyes should be using a `BasicDom`.